packages/web-runtime/src/pages/resolvePrivateLink.vue

```diff
@@ -15,18 +15,17 @@
       </template>
       <template v-else-if="errorMessage">
         <div class="oc-card-header oc-link-resolve-error-title">
-          <h2 v-if="isUnacceptedShareError">
-            {{ resource.name }}
-          </h2>
-          <h2 v-else key="private-link-error">
+          <h2 key="private-link-error">
             <span v-text="$gettext('An error occurred while resolving the private link')" />
           </h2>
         </div>
         <div class="oc-card-body oc-link-resolve-error-message">
           <p class="oc-text-xlarge">{{ errorMessage }}</p>
           <p
             v-if="isUnacceptedShareError"
-            v-text="$gettext('Note: You can reload this page after you accept the share.')"
+            v-text="
+              $gettext('Note: You can reload this page after you have enabled syncing the share.')
```

Comment:
```suggestion
              $gettext('You can reload this page after you have enabled syncing the share.')
```
Like it that way a bit more